src/app/App.js

```diff
@@ -1,11 +1,11 @@
 define([
+    './config',
```

Comment:
It's probably not that important for us as end users but for package authors when you use the name map can break. It was that issue I had with two dgrids. I think it's a best  practice.